cdap-client-tests/src/test/java/co/cask/cdap/client/MetadataHttpHandlerTestRun.java

```diff
@@ -755,6 +755,29 @@ public void testSystemMetadataRetrieval() throws Exception {
   }
 
   @Test
+  public void testExploreSystemTags() throws Exception {
```

Comment:
have a test case to show that a FileSet that isn't configured to be explorable - that it doesn't have the explore tag. With the current implementation in this PR, I think it will fail.